app/services/payment_provider_customers/gocardless_service.rb

```diff
@@ -38,24 +50,23 @@ def access_token
       organization.gocardless_payment_provider.access_token
     end
 
+    def client
+      @client || GoCardlessPro::Client.new(access_token: access_token, environment: :sandbox)
+    end
+
     def create_gocardless_customer
-      GoCardlessPro::Client
-        .new(access_token: access_token, environment: :sandbox)
-        .customers
-        .create(params: gocardless_create_payload)
+      client.customers.create(
+        params: {
+          email: customer.email,
+          company_name: customer.name,
+        }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)